app/screens/SpecialVotingScreen.tsx

```diff
@@ -110,39 +112,39 @@ const SpecialVotingScreen = ({ session }: { session?: Session }) => {
               ?.sort((a, b) => a.number - b.number)
               .map((result) => (
                 <View key={result.number} style={{ width: "100%" }}>
-                  <View style={AppStyles.halfWidth} >
-                    <T5 text={t("specialVoting.explanation") + "\n\n\n"} />
-                  </View>
-
-                  <SpacerV />
-                  <Text style={[AppStyles.center, AppStyles.b]}>{t("cfp.your_vote")}</Text>
-                  <View style={[AppStyles.containerHorizontalWrap, styles.voteContainer]}>
-                    <RadioButton
-                      label={t("specialVoting.collectively")}
-                      onPress={() => onVote(result.number, CfpVote.YES)}
-                      checked={votes?.[result.number] === CfpVote.YES}
-                      disabled={!canVote}
-                      loading={isSaving?.number === result.number && isSaving.vote === CfpVote.YES}
-                    />
-                    <RadioButton
-                      label={t("specialVoting.proportionally")}
-                      onPress={() => onVote(result.number, CfpVote.NO)}
-                      checked={votes?.[result.number] === CfpVote.NO}
-                      disabled={!canVote}
-                      loading={isSaving?.number === result.number && isSaving.vote === CfpVote.NO}
-                    />
-                    <RadioButton
-                      label={t("cfp.neutral")}
-                      onPress={() => onVote(result.number, CfpVote.NEUTRAL)}
-                      checked={(votes?.[result.number] ?? CfpVote.NEUTRAL) === CfpVote.NEUTRAL}
-                      disabled={!canVote}
-                      loading={isSaving?.number === result.number && isSaving.vote === CfpVote.NEUTRAL}
-                    />
-                  </View>
-
-                  <View style={AppStyles.halfWidth} >
-                    <T5 text={"\n\n\n" + t("specialVoting.description")} />
-                  </View>
+                  <T5 text={t("specialVoting.explanation") + "\n\n\n"} />
```

Comment:
just curious - why using 3x new lines instead of `<SpacerV height={x} />`?

Reply:
- No new View in ViewHierarchy to render/compute
- Wanted to keep the relative line height for the paragraph